xla_native_functions.yaml

```diff
@@ -52,6 +61,8 @@ supported:
   - convolution_overrideable
   - convolution_backward_overrideable
   - _copy_from
+  - _copy_from_and_resize
```

Comment:
Huh, not actually sure why these are showing up in the diff, since they're already there in master.

Reply:
fixed